/**:
  ros__parameters:
    # Dubins
    dubins_radius: 8.0
    dubins_step_size: 0.5
    replan_dist: 26.0
    no_of_laps: 100

    # Are these used?
    handover_offs: 1 # Meters before the end of the path to handover control of the vehicle
    
    # Linear speed limits
    max_speed         : 0.5 # m/s
    min_speed         : 0.0 # m/s
    # fw_speed: 0
    #origin_gps: [40.40229852, -86.84558228] # kepner
    #origin_gps: [40.448417, -86.867750]     # harner
    goal_lat          : 40.448417
    goal_lon          : -86.867750
    goal_head         : 1.5

    # Rotational PID gains
    kp : 1.0 # kp makes it faster
    kd : 70.0 # kd reduces overshoot
    ki : 0.5 # ki reduces settling error

    # ILOS
    replan_lookahead: 8.0  # Distance to look ahead to select next target waypoint when in need to replan. It is not replan_lookahead of ILOS.
    lookahead_min : 3
    lookahead_max : 20
    conv_rate     : 8.0
    gamma         : 0.015 # Integral gain

    sim_enable: true
